README.md

```
@@ -1,384 +1,377 @@
 
-## Contributors ✨
+## 👥 Contributors
 
-Thanks go to these wonderful people who have contributed to this project:
+A huge thank you to all our amazing contributors! 🎉
 
-<table>
+<div align="center">
+
+<a href="https://github.com/anurag2787/DateNow/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=anurag2787/DateNow" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)